### DISCUSSION TITLE
Different folders for different filetypes

### DISCUSSION THREAD

**Discussion by ravaq** [filecxx/FileCentipede#527] [Ideas]
hi
thanks for your superb DM!
It would be much better if there was an option to set different paths for different filetypes in the settings menu (similar to IDM and XDM). somebody may want to put, for example, videos in one drive and zip files in another drive, and it is a pain to transfer them manually.

thanks again

**Comment by ravaq** [comment]
Also there is no option to set software proxy to system proxy

**Comment by filecxx** [comment]
![image](https://user-images.githubusercontent.com/100348948/221369987-06210e9f-e6f5-4f9f-9ead-5ec05deba8a4.png)
![image](https://user-images.githubusercontent.com/100348948/221369998-c9ee31ec-307a-4aaf-bbb1-f6ee4938f534.png)
![image](https://user-images.githubusercontent.com/100348948/221370026-28b0f877-90d2-43f6-8d79-ce3564104417.png)

**Comment by ravaq** [comment]
thanks for response
but in start download dialog, the catalog isn't set automatically, and it must be set manually.

**Comment by filecxx** [comment]
![image](https://user-images.githubusercontent.com/100348948/221747646-e88b38cc-f04e-412a-aef9-b32ddae60a44.png)
![image](https://user-images.githubusercontent.com/100348948/221747740-c37e0af5-8d8f-434d-8e0d-9b671ab364f7.png)
![image](https://user-images.githubusercontent.com/100348948/221747854-e63d9e0c-d1e5-46ec-88ca-b0848a548f5a.png)
![image](https://user-images.githubusercontent.com/100348948/221747930-b3c0fc01-bd7b-4a46-9612-47d000cbc767.png)

**Reply by ravaq** [reply]
that is what I said, you need to set it in download window manually.

**Comment by ravaq** [comment]
that is what I said, you need to set it in download window manually.

**Comment by filecxx** [comment]
No, you don't, but you have to 
![image](https://user-images.githubusercontent.com/100348948/221895080-9d35ee69-3dd1-476e-b164-376aa6f6a806.png)
